index.html

```diff
@@ -37,304 +37,179 @@
-          premultiplication is lossy. It is not specifically intended to make
-          ImageData the "hub" of conversion, but it is the appropriate format
-          when the image pixel data needs to be read or modified.
+          The <dfn>create()</dfn>
         </p>
       </section>
       <section>
-        <h2>
-          Asynchronous getImageData
-        </h2>
+        <h3>
+          static <code>canDecodeType()</code> method
+        </h3>
         <p>
-          One alternative solution would be to add an asynchronous
-          <code>getImageData</code> method to the 2d context. However most
-          modern implementations use GPU-accelerated 2d contexts and such a
-          method could require readback from the GPU, which has costly
-          implications for performance. Use of a canvas also likely introduces
-          additional memory use and possibly intermediate copies of the image
-          data, which ought to be avoided for best performance and minimal
-          memory overhead. GPU resources may also be premultiplied, which is
-          lossy.
+          To check if the user agent <dfn>can decode an image type</dfn>, run
+          the steps below.
         </p>
+        <ol>
+          <li>...
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          static <code>canEncodeType()</code> method
+        </h3>
         <p>
-          Further, the logical counterpart to an asynchronous
-          <code>getImageData</code> method is an asynchronous
-          <code>putImageData</code> method. It is hard to see how this could be
-          implemented without creating race conditions if more synchronous
-          drawing happens during the processing. Should drawing done while it
-          is processing be queued up to happen after, or should the drawing be
-          overwritten when <code>putImageData</code> completes? Neither
-          solution seems great, and the proposed methods circumvent the canvas
-          entirely, side-stepping this problem.
+          To check if the user agent <dfn>can encode an image type</dfn>, run
+          the steps below.
         </p>
+        <ol>
+          <li>...
+          </li>
+        </ol>
       </section>
       <section>
-        <h2>
-          Canvas-in-workers
-        </h2>
+        <h3>
+          Checking supported operations
+        </h3>
         <p>
-          Another possible solution is to run the canvas in a worker. Then the
-          synchronous use of the canvas for conversion purposes will be
-          processed in parallel to the main thread. However to implement this
-          efficiently requires that various types be transferrable, which has
-          further implications. It also does not help convert HTMLImageElement
-          since it is a DOM element. Also it would likely require intermediate
-          copies of the image data or the additional memory overhead of the
-          canvas.
```

Comment:
While I agree the above (and sections above that as well) don't belong in a specification, maybe it should be moved to the explainer?